packages/xo-server-sdn-controller/src/index.js

```diff
@@ -844,6 +1269,23 @@ class SDNController extends EventEmitter {
 
   // ---------------------------------------------------------------------------
 
+  _getPoolNetwork(poolRef, crossPoolNetwork) {
+    const xapi = find(this._xapis, xapi => xapi.pool.$ref === poolRef)
+    const networkRef = find(crossPoolNetwork.networks, networkRef => {
+      try {
+        const network = xapi.getObjectByRef(networkRef)
```

Comment:
Inline.

Reply:
Not fixed.

Reply:
I'd pefer not to inline this.
I think it's more readable that way.